### DISCUSSION TITLE
Error 404 en mi perfil de github /   404 error on my github profile

### DISCUSSION THREAD

**Discussion by Lucas-Tejerina** [community/community#114265] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

![image](https://github.com/community/community/assets/163143585/8228e7e0-b984-4150-8fdb-4095b19d51fb)

 
Hello, I have a problem, when I try to access a friend's GitHub profile, either by link or by searching for their profile, this error appears :c

**Comment by Viniciusbianchini11** [comment]
Estou com o mesmo problema, consegiu resolver?

**Comment by tobykas** [comment]
I have the same issue, did you were able to solve it?
